packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -0,0 +1,284 @@
+import '@agoric/zoe/exported.js';
+import '@agoric/zoe/src/contracts/exported.js';
+import '@agoric/governance/exported.js';
+
+import { Far } from '@endo/marshal';
+import { E } from '@endo/eventual-send';
+import {
+  M,
+  makeScalarBigMapStore,
+  provideDurableMapStore,
+} from '@agoric/vat-data';
+import { AmountMath } from '@agoric/ertp';
+import {
+  atomicRearrange,
+  makeRatioFromAmounts,
+  makeRatio,
+  natSafeMath,
+  floorMultiplyBy,
+  provideEmptySeat,
+} from '@agoric/zoe/src/contractSupport/index.js';
+import { handleParamGovernance } from '@agoric/governance';
+import { makeTracer } from '@agoric/internal';
+import { FullProposalShape } from '@agoric/zoe/src/typeGuards.js';
+
+import { makeAuctionBook } from './auctionBook.js';
+import { BASIS_POINTS } from './util.js';
+import { makeScheduler } from './scheduler.js';
+import { auctioneerParamTypes } from './params.js';
+
+/** @typedef {import('@agoric/vat-data').Baggage} Baggage */
+
+const { Fail, quote: q } = assert;
+
+const trace = makeTracer('Auction', false);
+
+const makeBPRatio = (rate, currencyBrand, collateralBrand = currencyBrand) =>
+  makeRatioFromAmounts(
+    AmountMath.make(currencyBrand, rate),
+    AmountMath.make(collateralBrand, 10000n),
```

Comment:
we repeat this number a lot. worth importing from somewhere. maybe @agoric/internal or contractSupport